### ISSUE TITLE
Pasting and adding too long sentences result in unresponsive buttons ( android12)

### ISSUE THREAD

**Issue by robovoice1** [common-voice/sentence-collector#585]
Here is what i tried:
Copy and paste the cv faq from the cv website and add those very long sentencs (first recording edited sentences) into sentence collector. Second try and recording unedited sentences ( copy and paste as is)
Result by doing so is:
Unresponsive buttons (reject/skip/approve),
the too long sentences overlaying the buttons,
the first sentence stays visible and swiping for reject (left) the sentences behind the first get rejected (first recording)
Unresponsive confirm button. see pic.

![Screenshot_20220119-133807](https://user-images.githubusercontent.com/92784377/150140650-e564bd91-815e-4395-9e11-d568f755eac5.png)
![Screenshot_20220119-135414](https://user-images.githubusercontent.com/92784377/150140680-d7a52213-7166-4fbc-9ba5-04aaa574fb72.png)
![Screenshot_20220119-140103](https://user-images.githubusercontent.com/92784377/150140743-ab421d05-e289-4c65-9e61-e4486583e945.png)

Pixel 4 a 
Android 12/GrapheneOs
Fx mobile nightly android
Latest release

**Comment by robovoice1** [comment]
https://user-images.githubusercontent.com/92784377/150141829-ca6eace2-1d1d-4b1e-a466-3104af0b496d.mp4

**Comment by robovoice1** [comment]
https://user-images.githubusercontent.com/92784377/150142483-06aa48bc-ae27-428c-a2ba-6528b4b21119.mp4

**Comment by robovoice1** [comment]
Ps: Everything was rejected from my side and leftovers deleted from this "submission".

**Comment by robovoice1** [comment]
https://user-images.githubusercontent.com/92784377/150157131-f811045c-70e4-4e49-90ed-480be2e3bde2.mp4

Recording from Fairphone 3+, Android 10, Fx mobile nightly latest release

**Comment by MichaelKohler** [comment]
Thanks for filing this. This is the review before the actual submission, right?

If so, the validations only happens once fully submitted. Therefore it will include all sentences, even those that are not fulfilling all criteria. That might be worth an improvement, but would need to be discussed further.

That being said, we should look again into not overflowing the cards. I thought I had added a fix for that, but might not be remembering correctly.

**Comment by robovoice1** [comment]
Yes it is. But 1 sentence got in, which i deleted under my sentences. English sentence in german section, when trying this on Fairphone3.
